### ISSUE TITLE
Error from XSLT activate, 0

### ISSUE THREAD

**Issue by sandraros** [abapGit/abapGit#5001]
To reproduce:
- Install the latest version of [abap2xlsx](https://github.com/sapmentors/abap2xlsx)
- Uninstall it (all objects are deleted)
- Install abap2xlsx again
- This error happens during the activation of objects:
  ![image](https://user-images.githubusercontent.com/34005250/135757673-3cc08ace-3162-4322-ac8a-f1061d201d49.png)
  > Error from XSLT activate, 0 
  > Import of object ZEXCEL_TR_SHEET failed 
  (concerned object is in files [zexcel_tr_sheet.xslt.source.xml](https://github.com/sapmentors/abap2xlsx/blob/master/src/zexcel_tr_sheet.xslt.source.xml) and [zexcel_tr_sheet.xslt.xml](https://github.com/sapmentors/abap2xlsx/blob/master/src/zexcel_tr_sheet.xslt.xml)
- If I activate manually the transformation, no error.

Another XML transformation `zexcel_tr_shared_strings` is activated without any problem.

So, maybe it's due to the fact that `zexcel_tr_sheet` is using the class `ZCL_EXCEL_WRITER_HUGE_FILE`, which maybe is not active yet when the transformation is activated?

**Comment by mbtools** [comment]
yes, the order of processing objects is not guaranteed. there's a missing feature: https://github.com/abapGit/abapGit/issues/3536

let's try this https://github.com/abapGit/abapGit/pull/5003, which works similarly to tables that depend on classes. 

Could you please add a test case (1 XSLT + 1 CLAS) to https://github.com/abapGit-tests/XSLT_ref?

**Comment by sandraros** [comment]
Thanks I can work on it next week-end.
